cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -388,7 +388,7 @@ func (p *program) Stop(_ service.Service) error {
 }
 
 func main() {
-	flag.Usage = func() { usageExit(0) }
+	// flag.Usage = func() { usageExit(0) }
```

Comment:
Also just noticed that you already showed the output in #393

Reply:
I've update the PR removing the lines instead of commenting them out. For reference this is the output of `-h`\`-help`\`--help` after this PR: 

```
./build/bin/darwin_amd64/amazon-cloudwatch-agent --help
Usage of ./build/bin/darwin_amd64/amazon-cloudwatch-agent:
  -aggregator-filter string
    	filter the aggregators to enable, separator is :
  -config string
    	configuration file to load
  -config-directory string
    	directory containing additional *.conf files
  -console
    	run as console application (windows only)
  -debug
    	turn on debug logging
  -envconfig string
    	env configuration file to load
  -input-filter string
    	filter the inputs to enable, separator is :
  -input-list
    	print available input plugins.
  -output-filter string
    	filter the outputs to enable, separator is :
  -output-list
    	print available output plugins.
  -pidfile string
    	file to write our pid to
  -pprof-addr string
    	pprof address to listen on, not activate pprof if empty
  -processor-filter string
    	filter the processors to enable, separator is :
  -quiet
    	run in quiet mode
  -sample-config
    	print out full sample configuration
  -schematest
    	validate the toml file schema
  -section-filter string
    	filter the sections to print, separator is ':'. Valid values are 'agent', 'global_tags', 'outputs', 'processors', 'aggregators' and 'inputs'
  -service string
    	operate on the service (windows only)
  -service-display-name string
    	service display name (windows only) (default "Telegraf Data Collector Service")
  -service-name string
    	service name (windows only) (default "telegraf")
  -setenv string
    	set an env in the configuration file in the format of KEY=VALUE
  -test
    	enable test mode: gather metrics, print them out, and exit
  -test-wait int
    	wait up to this many seconds for service inputs to complete in test mode
  -usage string
    	print usage for a plugin, ie, 'telegraf --usage mysql'
  -version
    	display the version and exit
```